test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -489,6 +496,7 @@ define([
                     Jupyter.notebook = Mocks.buildMockNotebook();
                     spyOn(Jupyter.notebook, 'save_checkpoint');
                     testCase.cellId = `${testCase.state}-${testCase.action}`;
+                    testCase.noDataRefresh = true;
```

Comment:
These tests rely on having the jobs data set up in a particular way, so the data refresh should be skipped.